### ISSUE TITLE
selector-attribute-operator-space-after/before

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1138]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

All valid:
```css
a[target = _blank] {
    background-color: #ff0000;
}
```
```css
a[target= _blank] {
    background-color: #ff0000;
}
```
```css
a[target =_blank] {
    background-color: #ff0000;
}
```

**Comment by jeddy3** [comment]
Sgtm :)

- [ ] Name: `selector-attribute-punctuation-space-after`
- [ ] Name: `selector-attribute-punctuation-space-before`
- Primary options: `"always"|"never"`
- Secondary options: none
- Messages:
  - Expected: `Expected single space after/before "${punctuation}"`.
  - Rejected: `Unexpected whitespace after/before "${punctuation}"`.

**Comment by davidtheclark** [comment]
Is "operator" a better word than "punctuation"? Is there a word in the spec?

**Comment by evilebottnawi** [comment]
@jeddy3 i think @davidtheclark right. "operator" better than "punctuation"

**Comment by jeddy3** [comment]
> "operator" better than "punctuation"

SGTM. I did look through the specs before naming the rules, and it (rather tellingly) doesn't name the things. 

We use "operator" in [`media-feature-range-operator-space-after`](https://github.com/stylelint/stylelint/blob/master/src/rules/media-feature-range-operator-space-after/README.md), where the spec explicitly names the things.

And we use "punctuation" in [`media-feature-no-missing-punctuation`](https://github.com/stylelint/stylelint/blob/master/src/rules/media-feature-no-missing-punctuation/README.md), where we're referring to range operators *and* colons.

Lets rock with "operator" here, if only because it better aligns with the `*-before/after` rule of [`media-feature-range-operator-space-after`](https://github.com/stylelint/stylelint/blob/master/src/rules/media-feature-range-operator-space-after/README.md).

**Comment by evilebottnawi** [comment]
Merge. Close.

**Comment by ntwb** [comment]
Merged via https://github.com/stylelint/stylelint/commit/80816600231ec17f5c3d074adaf3ad77b51949ad
